specifyweb/frontend/js_src/lib/components/QueryBuilder/ResultsTable.tsx

```diff
@@ -169,16 +174,44 @@ function Row({
             className={`${getCellClassName(condenseQueryResults)} sticky`}
             role="cell"
           >
-            <Link.NewTab
-              className="print:hidden"
-              href={viewUrl}
-              rel="noreferrer"
-            />
+            {splitIds === undefined ? (
+              <Link.NewTab
+                className="print:hidden"
+                href={viewUrl}
+                rel="noreferrer"
+              />
+            ) : (
+              <Button.Icon
```

Comment:
add `className="print:hidden"` (to match the Link.NewTab above)

then can press Cmd+p in the browser to test how that look

or alternatively could emulate print mode using devtools - https://developer.chrome.com/docs/devtools/rendering/emulate-css#emulate_css_media_type_enable_print_preview